web/src/popover_menus.js

```diff
@@ -1180,6 +1180,11 @@ export function initialize() {
                 });
             });
 
+            $(".focus-dropdown").on("focus", (e) => {
+                e.preventDefault();
+                $popper.find("li:visible a").eq(0).trigger("focus");
+            });
```

Comment:
I don't quite understand what you're doing here, but I would expect keyboard UI for this menu to use the `popover_items_handle_keyboard` code that all of our other popovers use -- what's the thinking about that?

(I don't think `tab` behavior is important -- just match what we do with arrow keys in normal popovers)

Reply:
This question remains open; could be a follow-up.